tsparser/src/builder/mod.rs

```diff
@@ -182,3 +182,11 @@ impl std::fmt::Display for PlainError {
         write!(f, "{}", self.0)
     }
 }
+
+#[derive(Deserialize, Debug)]
```

Comment:
If you also derive Clone and Copy you don’t need to take references to the value